Frontend/android-native/app/src/main/res/layout/activity_main.xml

```diff
@@ -7,6 +7,7 @@
     android:id="@+id/main_layout"
     tools:context=".MainActivity">
     <include layout="@layout/splash" />
+     <include layout="@layout/referral_code" android:visibility="invisible"/>
```

Comment:
no need to include layout  in `activity_main.xml. you can directly inflate the layout using inflater.

Reply:
done